Rename  'Deposit' to 'Payout' (feature branch)

shendy-a8c opened the pull request:
Fixes https://github.com/Automattic/woocommerce-payments/issues/9595.

#### Changes proposed in this Pull Request
Part of project to rename `deposit` to `payout` -  paJDYF-fwb-p2.

This PR is to merge the feature branch [feature/deposits-payouts-rename](https://github.com/Automattic/woocommerce-payments/tree/feature/deposits-payouts-rename) that consists of renames from 'deposit' to 'payout'.

#### Testing instructions

* All unit tests should pass
* The PR is to merge a feature branch that has all individual PR's renaming specific portions of the WooPayments plugin UI. These changes have already been reviewed before those PR's were merged to this main feature branch. However, it will be good to take a glance to see if there is anything obvious missing / incorrect. 
* Check if the Changelog is appropriate or needs a change. 


-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

<!--
Make sure you edit the page for the current release when adding testing instructions.
We often create a blank page ahead of time for the next release.
If this PR need not be QA tested, edit to 'QA Testing Not Applicable'
-->

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

nagpai committed:
Rename  'Deposit' to 'Payout' in the transactions list component (#9546)

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>

botwoo commented:
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9606` or branch name `feature/deposits-payouts-rename` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=feature/deposits-payouts-rename) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 4967f4f24864ec24aaa5e65a5beff290ed11908b
- Build time: 2024-11-10 05:03:55 UTC

_Note: the build is updated when a new commit is pushed to this PR._

shendy-a8c committed:
Merge branch 'develop' into feature/deposits-payouts-rename

brucealdridge committed:
Rename deposits to payouts on the settings screen (#9532)

Co-authored-by: Rua Haszard <rua.haszard@automattic.com>
Co-authored-by: Shendy <73803630+shendy-a8c@users.noreply.github.com>

haszari committed:
rename "deposit" to "payout": instant deposit button & modal (#9554)

Co-authored-by: Rua Haszard <rua@automattic.com>
Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>
Co-authored-by: Shendy <73803630+shendy-a8c@users.noreply.github.com>

nagpai committed:
Payouts:  Rename 'deposit' to 'payout' in transaction details timeline (#9565)

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>

shendy-a8c committed:
Merge branch 'develop' into feature/deposits-payouts-rename

shendy-a8c committed:
Rename deposit to payout on the deposits list page. (#9528)

jessy-p committed:
Rename deposit to payout in account messaging. (#9611)

Co-authored-by: Jessy <jessy.pappachan@automattic.com>

jessy-p committed:
Merge branch 'develop' into feature/deposits-payouts-rename

jessy-p committed:
Rename Deposit to Payout in TOS modal (#9610)

Co-authored-by: Jessy <jessy.pappachan@automattic.com>

Jinksi committed:
Payments Overview – rename deposits to payouts (#9529)

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

jessy-p committed:
changed the URL and menu from Deposits to Payouts (#9530)

Co-authored-by: Jessy <jessy.pappachan@automattic.com>
Co-authored-by: Shendy <73803630+shendy-a8c@users.noreply.github.com>

jessy-p committed:
Rename instant deposits to instant payouts (#9622)

Co-authored-by: Jessy <jessy.pappachan@automattic.com>

jessy-p committed:
Rename deposits to Payouts in error messages (#9624)

Co-authored-by: Jessy <jessy.pappachan@automattic.com>

shendy-a8c committed:
Merge branch 'develop' into feature/deposits-payouts-rename

# Conflicts:
#	client/deposits/instant-payouts/index.tsx

nagpai committed:
Rename 'deposit' to 'payout' in 'Overview' section modals (#9623)

The PR is a part of a larger project renaming `deposit` to `payout` across the WooPayments plugin UI. 

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>

nagpai committed:
Rename 'deposit' to 'payout' in order note for fund withdrawal (#9632)

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>

shendy-a8c committed:
Rename 'deposit' to 'payout' in deposit details page (#9633)

shendy-a8c committed:
Merge branch 'develop' into feature/deposits-payouts-rename

nagpai committed:
Rename 'deposit' to 'payout' across doc urls (#9627)

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

nagpai committed:
Rename 'deposit' to 'payout'  in onboarding tasklist (#9642)

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>

shendy-a8c committed:
Merge branch 'develop' into feature/deposits-payouts-rename

shendy-a8c committed:
Rename deposit to payout in browser downloaded CSV filename and header. (#9643)

shendy-a8c committed:
Rename anchor link to the deposit schedule section of the payments settings (#9665)

shendy-a8c committed:
Rename 'deposit' to 'payout' on account connect / pre-onboarding page. (#9653)

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

Commit:
Add changelog

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

Commit:
Update tests

nagpai committed:
Rename 'deposit' to 'payout' in dev docs (#9669)

Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>
Co-authored-by: Eric Jinks <3147296+Jinksi@users.noreply.github.com>

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

shendy-a8c commented:
I see that we will have multiple changelog entries for this feature branch. IMO, we should just boil it down to just one changelog entry. What do you think @nagpai?

Maybe something generic like: Rename 'deposit' to 'payout'.

shendy-a8c commented on changelog/feature-deposits-payouts-rename:
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Rename 'deposit' to 'payout' across various places in the WooPayments admin UI.
```
To make this changelog file not to create a changelog entry:

```suggestion
Comment: Rename 'deposit' to 'payout' across various places in the WooPayments admin UI.

```

shendy-a8c commented on changelog/fix-9527-payouts-rename-settings:
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Payouts rename: Settings page, part of a larger change
```
```suggestion
Comment: Payouts rename: Settings page, part of a larger change

```

nagpai replied:
Thanks! Changed here - [80cae5a](https://github.com/Automattic/woocommerce-payments/pull/9606/commits/80cae5a7e928f559a235727b196a706286f25d37)

shendy-a8c commented on changelog/fix-9542-rename-payouts-instant-depost-modal-flow:
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Part of a larger change - renaming deposit to payout; this PR fixes instant payout modal + button
```
```suggestion
Comment: Part of a larger change - renaming deposit to payout; this PR fixes instant payout modal + button

```

shendy-a8c commented on changelog/feature-deposits-payouts-rename:
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Rename 'deposit' to 'payout' across various places in the WooPayments admin UI.
```
To make this changelog file not to create a changelog entry:

```suggestion
Significance: patch
Type: dev
Comment: Rename 'deposit' to 'payout' across various places in the WooPayments admin UI.

```

nagpai replied:
This is the main changelog for the PR and should be probably kept as it is?

shendy-a8c commented on changelog/fix-9542-rename-payouts-instant-depost-modal-flow:
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Part of a larger change - renaming deposit to payout; this PR fixes instant payout modal + button
```
```suggestion
Significance: patch
Type: dev
Comment: Part of a larger change - renaming deposit to payout; this PR fixes instant payout modal + button

```

nagpai replied:
Changed here, thank you! - [80cae5a](https://github.com/Automattic/woocommerce-payments/pull/9606/commits/80cae5a7e928f559a235727b196a706286f25d37)

shendy-a8c commented on changelog/fix-9585-rename-instant-payouts:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
Not sure if only dev or it can be any value here so that changelog is 'ignored'.

```suggestion
Type: dev
```

nagpai replied:
I think the type can be `update` .. It should be `patch` and have a `Comment`, to be ignored from the main changelog . 

The PR also delves into merchant-facing UI and hence logically should be `update` instead of `dev`.

shendy-a8c commented on changelog/fix-9588-rename-payouts-error-messages:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
```suggestion
Type: dev
```

nagpai replied:
Keeping unchanged based on my reply [here ](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295)

shendy-a8c commented on changelog/update-9524-deposit-payout-deposits-list:
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Change deposit to payout in deposits list page.
```
```suggestion
Significance: patch
Type: dev
Comment: Change deposit to payout in deposits list page.

```

nagpai replied:
Changed -  [80cae5a](https://github.com/Automattic/woocommerce-payments/pull/9606/commits/80cae5a7e928f559a235727b196a706286f25d37)

shendy-a8c commented on changelog/update-9543-transactions-list-deposit-to-payout-rename:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
```suggestion
Type: dev
```

nagpai replied:
Keeping unchanged for now based on my [earlier reply](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295).

shendy-a8c commented on changelog/update-9564-transaction-timeline-deposit-to-payout-rename:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
```suggestion
Type: dev
```

nagpai replied:
Keeping unchanged for now [ref](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295).

shendy-a8c commented on changelog/update-9578-79-rename-deposit-overview-modal:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
```suggestion
Type: dev
```

nagpai replied:
Keeping unchanged for now - [ref](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295) .

shendy-a8c commented on changelog/update-9581-rename-deposit-onboarding-tasklist:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
```suggestion
Type: dev
```

nagpai replied:
Keeping unchanged for now - [ref](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295).

shendy-a8c commented on changelog/update-9587-doc-url-payout-rename:
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```
```suggestion
Type: dev
```

nagpai replied:
Keeping unchanged for now - [ref](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295).

Commit:
Merge branch 'develop' into feature/deposits-payouts-rename

Commit:
Change constituent changelog to comments

Commit:
Add main changelog

shendy-a8c committed:
Merge branch 'develop' into feature/deposits-payouts-rename

jessy-p committed:
Add dismissable Notice to message the rename of Deposits to Payout (#9680)

Co-authored-by: Jessy <jessy.pappachan@automattic.com>
Co-authored-by: Nagesh Pai <nagesh.pai@automattic.com>
Co-authored-by: Shendy <73803630+shendy-a8c@users.noreply.github.com>

shendy-a8c commented:
Changelog looks good. Tested well on JN site. I can't approve my own PR...

shendy-a8c commented:
Bypassing branch rule that requires approval. Merged the PR.
